tech/languages/python/multiple-pythons.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Multiple Pythons
+subsection: python
+order: 8
+---
+
+# Multiple Python Interpreters
+
+If you are working on a piece of Python software, you probably want to test it
+on multiple Python interpreters. On Fedora, that's easy, all you have to do is
+use `dnf` to install whta you need. Currently Fedora has the following Pythons
+ready for you in the repositories:
+
+ * CPython 3.6
+ * CPython 3.5
+ * CPython 3.4
+ * CPython 3.3
+ * CPython 2.7
+ * CPython 2.6
+ * PyPy
+ * PyPy 3
+ * Jython
+ * MicroPython
+
+Quite a nest, isn't it?
+You can install them like this:
+
+```bash
+sudo dnf install python33  # to get CPython 3.3
+sudo dnf install python34  # to get CPython 3.4
+sudo dnf install python26  # to get CPython 2.6
+sudo dnf install pypy pypy3 jython python35  # to get more at once
+```
+
+After that, you can run an interactive console or your script with let's say
+CPython 3.4:
+
+```bash
+python3.4
+```
+
+Or get pip and use it to install packages:
+
+```bash
+python3.4 -m ensurepip --user
```

Comment:
Right, with a mention that there are no Fedora packages for the alternative Pythons.